### ISSUE TITLE
Show relative path in peek

### ISSUE THREAD

**Issue by finalpatch** [emacs-lsp/lsp-ui#261]
Right now lsp-ui-peek shows full path, so the file name part is usually truncated by the window border.  I can only see the first half of the path to every file, which are all identical,  and doesn't provide any value.

I suggest change this to show path relative to project root.  This will make the path a lot more useful and waste less space.

**Comment by finalpatch** [comment]
Or at least provide a customize option to show relative path

**Comment by yyoncho** [comment]
are the files under the current project root?

**Comment by finalpatch** [comment]
Yes they are in a git repository, which is considered a project root. I don't use projectile though.

**Comment by c02y** [comment]
I think relative path is not enough, for example, I'm using Rust, the path in the peek would be something like 
```
/home/user/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/time.rs
```
The path is so long, when I use multiple windows in one frame, I only see a small part of it, I have to maximize the current window to get the full path.

**Comment by tonicebrian** [comment]
Does someone has a solution for this?
